qa/rolling-upgrade/build.gradle

```diff
@@ -42,6 +42,8 @@ for (Version bwcVersion : BuildParams.bwcVersions.wireCompatible) {
     }
   }
 
+  String oldVersion = bwcVersion.toString().replace('-SNAPSHOT', '')
```

Comment:
I don't think bwc versions will ever contain `-SNAPSHOT`, do they?

Reply:
I stole this from some other code. I'll check. If it doesn't I'll remove this and then zap it from the other spot too.

Reply:
Yeah - it doesn't have -SNAPSHOT. I've pushed an update that drops it from this PR and from the place where I found it.